hack/kustomize/overlays/test/collector.yaml.template

```diff
@@ -0,0 +1,44 @@
+clusterName: YOUR_CLUSTER_NAME
+enableEvents: false
+enableDiscovery: true
+useClassicPrometheusParser: USE_CLASSIC_PROMETHEUS
+flushOnce: FLUSH_ONCE
+flushInterval: FLUSH_INTERVALs
+defaultCollectionInterval: COLLECTION_INTERVALs
+
+
+sinks:
+- proxyAddress: wavefront-proxy.NAMESPACE.svc.cluster.local:2878
```

Comment:
redirectToLog needs to be templatized as well. It is needed as part of flushOnce.